hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieSchemaUtils.scala

```diff
@@ -45,6 +45,8 @@ import scala.collection.JavaConverters._
 object HoodieSchemaUtils {
   private val log = LoggerFactory.getLogger(getClass)
 
+
+
```

Comment:
nit: please remove the unnecessary newline. Also, let's move the schema util methods in `HoodieSparkSchemaUtils` to this object - see https://github.com/apache/hudi/pull/12623/files#r1926327306